website/views.py

```diff
@@ -98,11 +97,6 @@ def _render_nodes(nodes, auth=None, show_path=False):
 
 @collect_auth
 def index(auth):
-    """Redirect to dashboard if user is logged in, else show homepage.
-
-    """
-    if auth.user:
-        return redirect(web_url_for('home'))
     return {}
```

Comment:
So if you wanted to you could remove the `index` view function entirely, pass `{}` to `Rule`, and then update the places where we call `web_url_for('index')` to call `web_url_for('')` instead. But I'm not going to block this PR on that. :-)